test/Microsoft.AspNet.Mvc.Razor.Test/RazorViewTest.cs

```diff
@@ -359,14 +363,11 @@ public class RazorViewTest
         public async Task RenderAsync_ExecutesLayoutPages()
         {
             // Arrange
-            var expected =
-@"layout-content
-head-content
-body-content
-foot-content";
+            var expected = @"layout-content&#xD;&#xA;head-content&#xD;&#xA;body-content&#xD;&#xA;foot-content";
```

Comment:
Were these newline characters supposed to be written as literals instead of being run through the encoder?